homeassistant/components/spider/const.py

```diff
@@ -0,0 +1,6 @@
+"""Constants for the Spider integration."""
+
+DOMAIN = "spider"
+DEFAULT_SCAN_INTERVAL = 300
+
+PLATFORMS = ("climate", "switch")
```

Comment:
```suggestion
PLATFORMS = ["climate", "switch"]
```

Usually we use a list here.